orc8r/gateway/python/magma/magmad/rpc_servicer.py

```diff
@@ -48,7 +51,7 @@ def __init__(self,
                  service_manager: ServiceManager,
                  mconfig_manager: MconfigManager,
                  command_executor: CommandExecutor,
-                 loop):
+                 loop, print_grpc_payload: bool = False):
```

Comment:
Would be good to insert a new line after `loop,` for readability
(And if you feel like it, do you mind adding type annotation for loop?)